app/formats.js

```diff
@@ -0,0 +1,31 @@
+export default {
+  time: {
+    hhmmss: {
+      hour: 'numeric',
+      minute: 'numeric',
+      second: 'numeric',
+    },
+  },
+  date: {
+    hhmmss: {
+      hour: 'numeric',
+      minute: 'numeric',
+      second: 'numeric',
+    },
+  },
+  number: {
+    compact: { notation: 'compact' },
+    EUR: {
+      style: 'currency',
+      currency: 'EUR',
+      minimumFractionDigits: 2,
+      maximumFractionDigits: 2,
+    },
+    USD: {
+      style: 'currency',
+      currency: 'USD',
+      minimumFractionDigits: 2,
+      maximumFractionDigits: 2,
+    },
+  },
+};
```

Comment:
This is the new way to configure internationalization. Instead of having a sub-field in the ember config file, there's this file and also an ember-intl.js in the config/folder.